### PR TITLE
Billing plan

### PR DESCRIPTION
**Pull request by bantic**
Refs #291 

  * [x] add /organizations/:id/billing/plan route
  * [x] style the plan panels on plan page (can we use flexbox?)
  * [ ] ~~display billing overview header (plan name, payment method, invoice date, etc: http://take.ms/kNA55)~~
  * [x] wire the UI to update the organization's plan when requested
  * [x] display a confirmation modal when upgrading
  * [x] Auth: add backend code that handles upgrading an organization (WIP: https://github.com/aptible/auth.aptible.com/pull/151)
  * [x] display error if upgrade fails
  * [x] track an event, show a modal, when a user clicks "ask about our hipaa compliant plans" button
  * [x] change confirmation modal as explained in #291 

@mixonic for your review cc @sandersonet 

@sandersonet : I've used mostly ad-hoc CSS in `app/styles/billing-plans.scss`. Feedback there welcomed and/or you or I can address styling improvements after merge. These seem to be about 80-90% of the way towards matching the mocks. A notable exception is that I didn't fix the height of the panels, so when the window changes they can get out of sync, height-wise. I've added screenshots in comments below of most of the styled UI.

I am going to punt on displaying the billing overview header for now, as it relies on API calls that we do not yet have.

### CODE REVIEW

**Comment by bantic** [comment]
Styled UI for platform plan: 
image

**Comment by bantic** [comment]
styled UI when plan = development:

image

**Comment by bantic** [comment]
displayed confirmation modal:

image

**Comment by bantic** [comment]
error message displayed when upgrade fails:

image

**Comment by bantic** [comment]
account upgrade request modal:

image

File: app/organization/billing/plan/route.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    organization.set('plan', planType);
+
+    controller.set('error', null);
+    controller.set('isUpgrading', true);
+    organization.save().catch((e) => {
+      controller.set('error',
+                     Ember.get(e, 'responseJSON.message'));
+      organization.rollback();
+    }).finally(() => {
+      this.controller.set('isUpgrading', false);
```
Comment (mixonic) [inline app/organization/billing/plan/route.js]: already plucked it off, and used without `this.` on line 44

File: app/styles/billing-plans.scss [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+  .plan-include-subheader {
+    color: $color-light;
+    font-size: smaller;
+  }
+
+  .plan-btn {
+    width: 100%;
+
+    &[disabled] {
```
Comment (mixonic) [inline app/styles/billing-plans.scss]: hm, I would have expected you want the psuedo selector of `&:disabled`, but if the attribute selector works, it works.

**Comment by mixonic** [comment]
looking fantastic. :+1:, will need some UI polish before production.

It would be simple to wrap this in a FF so master isn't blocked from deploy while the styling gets handled.

File: app/styles/billing-plans.scss (hunk 2 above)
Comment (bantic) [inline app/styles/billing-plans.scss]: that seems like a more normative way to do it, I agree. I lifted this from https://github.com/aptible/aptible-sass/blob/c53eb6d47f9877bdac59b144aac3cccf407dcc4c/dist/styles/aptible_buttons.scss#L12. I'll update to use the pseudo selector.

**Comment by bantic** [comment]
Added an FF to hide the "Billing" link in the organization settings sidebar. False in production, true otherwise.

**Comment by mixonic** [comment]
:fireworks: :fireworks: :fireworks: :fireworks: :fireworks: :fireworks: :fireworks: :fireworks:

**Pull request merged**